webapp/src/components/sidebar_right/sidebar_right.jsx

```diff
@@ -102,7 +102,11 @@ export default class SidebarRight extends React.PureComponent {
 
     render() {
         const baseURL = this.props.enterpriseURL ? this.props.enterpriseURL : 'https://github.com';
-        const orgQuery = this.props.org ? '+org%3A' + this.props.org : '';
+        let orgQuery = '';
+        this.props.orgs.map((org) => {
```

Comment:
The `PropType` for this should be marked as required since we assume that it's defined here